### PR TITLE
ElasticSearch connect

### PR DESCRIPTION
**Pull request by brachi-wernick**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -258,4 +259,98 @@ as well and select the left and right values wisely - simple and small as possib
...
+       "schema.ignore": "true"
+    
+     } 
+    }
+    ```
+    
+    This request will send the data of topic `users-counts` directly to Elasticsearch, and will create an index with the topic name.
+
+4. To make sure connector created successfully, run: `curl -s -X GET http://localhost:8083/connectors/elk-connect/status`
+You must get success result:
```
Comment (hcloli) [inline README.md:305]: Use the word "should" instead of "must": 
"You should get success result:"

File: connect/pom.xml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    <parent>
+        <artifactId>kafka-stream-skeleton</artifactId>
+        <groupId>com.tikal.org</groupId>
+        <version>1.0-SNAPSHOT</version>
+    </parent>
+
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>connect</artifactId>
+
+    <repositories>
```
Comment (hcloli) [inline connect/pom.xml:14]: Brachi, I think this repository is unnecessary. Try to remove the "repositories" section and see if all works well.

**Review by hcloli — APPROVED** [review]

File: connect/pom.xml (hunk 2 above)
Comment (brachi-wernick) [reply connect/pom.xml:14]: no. it's not in maven central repository
```
[ERROR] Failed to execute goal on project connect: Could not resolve dependencies for project com.tikal.org:connect:jar:1.0-SNAPSHOT: Failure to find com.github.confluentinc:kafka-connect-elasticsearch:jar:3.3.1 in https://repo.maven.apache.org/maven2 was cached in the local repository, resolution will not be reattempted until the update interval of central has elapsed or updates are forced -> [Help 1]

```

Comment (hcloli) [reply connect/pom.xml:14]: OK. Thanks.

**Pull request merged**